src/buildHTML.js

```diff
@@ -696,22 +697,26 @@ groupTypes.accent = function(group, options) {
         // shift the accent over to a place we don't want.
         accent.italic = 0;
 
+        accentBody = makeSpan(["accent-body"], [accent]);
+
+        // CSS defines `.katex .accent .accent-body { width: 0 }`
+        // so that the accent doesn't contribute to the bounding box.
+        // We need to shift the character by its width (effectively half
+        // its width) to compensate.
+        let left = -accent.width / 2;
+
+        // Shift the accent over by the skew.
+        left += skew;
+
         // The \H character that the fonts use is a combining character, and
-        // thus shows up much too far to the left. To account for this, we add a
-        // specific class which shifts the accent over to where we want it.
+        // thus shows up much too far to the left. To account for this, we add
+        // a manual shift of the width of one space.
         // TODO(emily): Fix this in a better way, like by changing the font
-        let accentClass = null;
         if (group.value.label === '\\H') {
-            accentClass = "accent-hungarian";
+            left += 0.5;  // width of space
         }
 
-        accentBody = makeSpan([], [accent]);
-        accentBody = makeSpan(["accent-body", accentClass], [accentBody]);
-
-        // Shift the accent over by the skew. Note we shift by twice the skew
-        // because we are centering the accent, so by adding 2*skew to the left,
-        // we shift it to the right by 1*skew.
-        accentBody.style.marginLeft = 2 * skew + "em";
+        accentBody.style.left = left + "em";
```

Comment:
I found that change the `left` value in the browser didn't actually change the position of the accent body.  It appears that `.katex .accent>.vlist-t` is the style that's actively centering accents.  Maybe we don't need the glyph width after all.

Reply:
It's true that `vlist` is doing horizontal centering.  That's how accents were able to work before.  The catch is that we need to set the accent to zero width so that it doesn't affect the bounding box.  When that happens, the left edge of the accent gets centered, and the accent itself extends to the right.  Hence the need for the horizontal offset.

I do wonder whether there's a way to do this with CSS, without needing the width of the character...